NSPortExample/MessageSender/MessageSender/MainWindowController.cs

```diff
@@ -6,10 +6,11 @@
 
 namespace MessageSender
 {
-	public partial class MainWindowController : NSWindowController
+	public partial class MainWindowController : NSWindowController,INSTextFieldDelegate
 	{
 		CFMessagePort msgPort;
-
+		object sender;
+		EventArgs e;
```

Comment:
You can get rid of `sender` and `e` declaration and call `SendMessage` like this:
```
SendMessage (null, null);
```
on line 56.